assets/js/modules/analytics/components/common/PropertySelect.js

```diff
@@ -91,9 +91,16 @@ export default function PropertySelect() {
 					<Option
 						key={ index }
 						value={ id }
-						data-internal-id={ internalWebPropertyId } // eslint-disable-line sitekit/acronym-case
+						data-internal-id={ internalWebPropertyId } /* eslint-disable-line sitekit/acronym-case */
```

Comment:
Please use `//` comments for single lines as it was before (was this maybe changed on accident?
```suggestion
						data-internal-id={ internalWebPropertyId } // eslint-disable-line sitekit/acronym-case
```